code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -213,6 +214,7 @@
 	if(recipient && !QDELETED(recipient))
 		recipient.gain_health(heal_amount)
 		to_chat(recipient, SPAN_XENONOTICE("We recover a bit from our injuries, and begin to regenerate rapidly."))
+		to_chat(bound_xeno, SPAN_XENOHIGHDANGER("One of our picked resin fruits has been consumed."))
```

Comment:
This isn't the correct place to put this. This means only `/obj/effect/alien/resin/fruit/greater` will indicate they are consumed.

In other words healing fruit now has the consume message, but no other fruit does. Instead this message ought to be in either `finish_consume` or `delete_fruit` or `Destroy`

My recommendation is to make these changes: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/a0c9e01f-6576-41f4-9c5d-04294e98fc56)

```suggestion
```